### ISSUE TITLE
Local builds sometimes gets stuck when there's a task failure

### ISSUE THREAD

**Issue by kenotron** [OfficeDev/office-ui-fabric-react#10254]
Reported by @KevinTCoughlin:

 master build is consistently failing for me on my workstation with yarn build. The failures occur with a11y-tests. I have not been able to build master successfully since lerna migration. Yesterday I observed that "zombie" processes are leftover for the OS to kill after some time. Those processes pin my CPU to 100%.

![image](https://user-images.githubusercontent.com/34725/63619067-8cda4780-c5a2-11e9-8059-cdcece1860f4.png)

**Comment by kenotron** [comment]
I'll attempting to repro - this is run inside an admin powershell window... i'll try that

It would help if get more info - puppeteer seems to have issues getting to communicate with chrome. It would help to see if this occurs on the same machine on a clean clone on a normal cmd.exe window.

**Comment by kenotron** [comment]
Notes from @KevinTCoughlin: 

repro steps: 

clone master OUFR
introduce a lint break in one of the packages
run yarn build
watch ProcExp

**Comment by KevinTCoughlin** [comment]
Thanks Ken, I'll try to get a repro branch for you. I have seen this when any individual project build fails (returns a non-zero exit code).

**Comment by kenotron** [comment]
The recent changes of the forceExit will address that issue that @KevinTCoughlin originally reported, but it is spitting an ugly red warning message. To get rid of that, I'd essentially have to write my own "CLI" or jest task function that wraps the node API call to jest. This is doable but I hadn't done that yet. I'll likely address it on another issue tho. I'm closing this one to clean up the queue :)
